### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by mdxprograms** [yujinakayama/atom-auto-update-packages#25]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:57:13)
Object.activate (/Users/josh/.atom/packages/auto-update-packages/lib/auto-update-packages.coffee:21:9)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30)
```

**Comment by iamstarkov** [comment]
:+1:

**Comment by tdbs** [comment]
:+1:

**Comment by yujinakayama** [comment]
Fixed in auto-update-packages 1.0.0. (#31)

**Comment by mdxprograms** [comment]
is this going to be pushed to the Atom 1.0 release?
